pinot-core/src/main/java/com/linkedin/pinot/core/realtime/impl/RealtimeSegmentImpl.java

```diff
@@ -695,4 +698,12 @@ public void setSegmentPartitionConfig(SegmentPartitionConfig segmentPartitionCon
   public SegmentPartitionConfig getSegmentPartitionConfig() {
     return segmentPartitionConfig;
   }
+
+  public void setStarTreeIndexSpec(StarTreeIndexSpec starTreeIndexSpec) {
```

Comment:
the methods to set and get start tree index spec in this class don't seem to be used. Or, did I read it wrong?